artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/ActiveMQServerImpl.java

```diff
@@ -1109,9 +1109,7 @@ void stop(boolean failoverOnServerShutdown, final boolean criticalIOError, boole
       storageManager = null;
 
       sessions.clear();
-
-      activateCallbacks.clear();
```

Comment:
This will make ClearActivateCallbackTest to fail

Reply:
@mtaylor you basically are trying to revert 36fc14a30d6f1128c525c91cc9f8752fdd3f540c / ARTEMIS-1704

if you do revert this, please remove the test.

@jbertram what is the intent, and why @mtaylor is reverting this?

Reply:
this is also failing checkstyle. there's an extra space somewhere.